pcs/lib/commands/booth.py

```diff
@@ -629,15 +629,23 @@ def _ticket_operation(
     _ensure_live_env(env, booth_env)
 
     if not site_ip:
-        site_ip_list = resource.find_bound_ip(
-            get_resources(env.get_cib()), booth_env.config_path
-        )
-        if len(site_ip_list) != 1:
+        try:
+            site_ip_list = resource.find_bound_ip(
+                get_resources(env.get_cib()), booth_env.config_path
+            )
+            if len(site_ip_list) != 1:
+                raise LibraryError(
+                    ReportItem.error(
+                        reports.messages.BoothCannotDetermineLocalSiteIp()
+                    )
+                )
+        except LibraryError as e:
             raise LibraryError(
-                ReportItem.error(
-                    reports.messages.BoothCannotDetermineLocalSiteIp()
+                    ReportItem.error(
+                        reports.messages.BoothCannotDetermineLocalSiteIp()
+                    )
                 )
-            )
+
```

Comment:
Please put this BoothCannotDetermineLocalSiteIp LibraryError into variable `site_error` because it is used twice and use `try .. except` block only for getting cib:
```
try:
    cib = env.get_cib()
except LibraryError as e:
    raise site_error from e
```

Please do the same for pcs-0.10.